Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -241,7 +253,19 @@ bool NANDImporter::ExtractCertificates()
 
     const std::string pem_file_path = m_nand_root + std::string(certificate.filename);
     const ptrdiff_t certificate_offset = std::distance(content_bytes.begin(), search_result);
+    if (certificate_offset < 2 || static_cast<size_t>(certificate_offset) >= content_bytes.size())
```

Comment:
```
const s32 min_offset = 2;
if (certificate_offset < min_offset || static_cast<size_t>(certificate_offset) >= content_bytes.size())
```

then `min_offset` can be used below.

Reply:
I'm also realizing right now that the `static_cast<size_t>(certificate_offset) >= content_bytes.size()` check is redundant, `std::search` cannot return an iterator past the end of `content_bytes` except in the not-found case, which is already checked.

Reply:
Oops, I completely missed that...